src/Domain/CoordinateMaps/Wedge2D.cpp

```diff
@@ -74,19 +124,21 @@ Wedge2D::jacobian(const std::array<T, 2>& xi) const noexcept {
        &dxdeta = get<0, 1>(jacobian_matrix),
        &dydxi = get<1, 0>(jacobian_matrix);
 
-  dxdxi = -0.5 * inner_radius_ / sqrt(2.0) + 0.5 * outer_radius_ * cos(theta);
-  dxdeta = -0.5 * (1 + xi[0]) * outer_radius_ * sin(theta) * M_PI_4;
-  dydxi = -0.5 * inner_radius_ * xi[1] / sqrt(2.0) +
-          0.5 * outer_radius_ * sin(theta);
-  dydeta = 0.5 * (1 - xi[0]) * inner_radius_ / sqrt(2.0) +
-           0.5 * (1 + xi[0]) * outer_radius_ * cos(theta) * M_PI_4;
-  if (positioning_of_wedge_.axis() == Direction<2>::Axis::Eta) {
+  dxdxi = -0.5 * inner_radius_ / sqrt(2.0) + 0.5 * outer_radius_ * one_over_rho;
```

Comment:
move all potential DataVectors (i.e. ReturnTypes) as far right as possible in the math